tags/whyjava8.md

```diff
@@ -4,5 +4,6 @@ embed:
   title: Why does Prism Launcher ask me to change Java version?
   color: orange
 ---
+Minecraft versions before 1.17 required Java 8 and have issues with newer java, while newer versions require Java 17, so you need to change Java version. 
 
-Minecraft versions before 1.17 required Java 8 and have issues with newer java, while newer versions require Java 17, so you need to change java version. Some people think Java 8 is very outdated, but it's actually an LTS, it's still getting updates. If one of your mods is weird and requires newer java, you can bypass this going to instance settings-Java and ticking 'Skip java compatibility checks', but be aware of potential issues, such as [random CMEs](https://bugs.mojang.com/browse/MC-149777).
+If one of your mods is weird and requires newer java, you can bypass this going to instance settings-Java and ticking 'Skip java compatibility checks', but be aware of potential issues, such as [random CMEs](https://bugs.mojang.com/browse/MC-149777).
```

Comment:
I don't know if it's a good idea to remove this:
> Some people think Java 8 is very outdated, but it's actually an LTS, it's still getting updates.

Reply:
I don't see people complaining saying that Java 8 is outdated all that often so I figured I would just remove it. I can toss it back in if needed.

Reply:
Readded